03-task-manager/starter/controllers/tasks.js

```diff
@@ -0,0 +1,57 @@
+const Task = require('../models/task')
+const asyncWrapper = require('../middleware/async')
+const { createCustomError } = require('../errors/custom-error')
+const getAllTasks = (req, res) => {
+    res.send('all items from the file')
```

Comment:
The `getAllTasks` function needs to return all of the tasks from the API!

You can use the `find` method on the `Task` model to achieve this.
[https://mongoosejs.com/docs/api/model.html#Model.find()](https://mongoosejs.com/docs/api/model.html#Model.find())